src/main/java/com/google/flourbot/entity/EntityModuleImplementation.java

```diff
@@ -96,15 +98,27 @@ public EntityModuleImplementation(DataStorage dataStorage) {
 
         switch(sheetAction) {
           case ("Sheet Append Action"):
-            action = new SheetAppendRowAction(columnValue, sheetAction, sheetUrl);
+            action = new SheetAppendRowAction(sheetUrl, columnValue);
             break;
  
           case ("Read Row Action"):
-            action = new SheetReadRowAction(columnValue, sheetAction, sheetUrl);
+            int row = 1; // TODO: get it from database
```

Comment:
Are you planning on making a new class for these queries related to spreadsheet appends/reads, or just make individual functions in this class for the queries?

Reply:
Each action is a different subclass of SheetAction, and so the subclasses add in methods and attributes that are more specific to what they handle